src/callbacks/functions/enableDisableControls.js

```diff
@@ -0,0 +1,10 @@
+export default function enableDisableControls() {
```

Comment:
👍 Good to modularize this. 

Looking at it again, I think I would just hide the controls (instead of disabling) on the participant view. I don't think it really provides any extra info by having them visible ...  Can make an issue in v1.1 instead of doing this in 1.0 if it's going to take more than a few minutes.

Reply:
In fact, looks like some of the filters are just ignored (site, grouping), so I'd definitely vote for hide instead of disable.

Reply:
I like the idea of hiding controls to clean up the participant timelines.  I do think event-level filters should remain.  Down the line let's add variable metadata to the filters like we've done in the AE table.

Reply:
yeah, good idea. Make an issue for future work? For now, I'll leave it to you whether we should hide or just disable items from the `filters` array - I guess disable might make more sense as a stop gap?